ivy/functional/backends/jax/creation.py

```diff
@@ -20,14 +22,14 @@
 
 
 def arange(
-    start,
-    stop=None,
-    step=1,
+    start: Number,
```

Comment:
`float` encapsulates `int`, and so we can just use `float` for all such type hints, rather than `Number`